scripts/flattener_mods/gather.py

```diff
@@ -197,6 +197,15 @@ def gather_metdata(obj_type, properties, values_to_add, objs, connection):
 			latkey = (obj_type + '_' + prop).replace('.', '_')
 			key = constants.PROP_MAP.get(latkey, latkey)
 			values_to_add[key] = value
+		elif prop == 'dbxrefs':
```

Comment:
get_value() is actually returning constants.UNREPORTED_VALUE ("unknown") when there is no dbxrefs. This is the expected behavior. And what is happening is that it is not finding the constants.ACCEPTED_ACCESSIONS in the string "unknown", so it is joining an empty list, which ends up being an empty string. 
So, you'll need to add an if statement at 201:
`if value != constants.UNREPORTED_VALUE:`
And that way you won't need to make any changes to line 592.